app/assets/stylesheets/pregnancies.scss

```diff
@@ -2,6 +2,9 @@
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
 
+$dcaf-color: #5D3E8D;
+$font-size: 17px;
```

Comment:
It's a bummer to me that rails doesn't pass these variables down from the application stylesheet, but I googled for a better way and saw nothing but nerds yelling at each other, haha. We shouldn't make any changes now, but if we anticipate a lot of SCSS editing in the future we may want to look into better ways to do this.